tests/mocks/redis_sessions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Mock redis sessions db
+"""
+import pytest
+
+session_data = {}
```

Comment:
⛏️ 📝 You can use a singleton or Borg pattern for shared instance state.

Reply:
Yes needs more work I think, but is functional for now - would love a demo